core/kotlinx-coroutines-core/src/test/kotlin/kotlinx/coroutines/experimental/channels/SubscribableVariableTest.kt

```diff
@@ -0,0 +1,62 @@
+package kotlinx.coroutines.experimental.channels
+
+import kotlinx.coroutines.experimental.*
+import org.hamcrest.core.IsEqual
+import org.junit.Assert.assertThat
+import org.junit.Test
+
+class SubscribableVariableTest : TestBase() {
+
+    @Test
+    fun testBasicScenario() = runBlocking {
+        expect(1)
+        val subscribable = SubscribableVariable(42)
+        var variable by subscribable
```

Comment:
can you use`val`?

Reply:
Here no, because at line 26 we do `variable = 24`. But `testObservableValue` use a `val`.